README.md

```diff
@@ -72,7 +72,7 @@ Download and build OpenDataPlane (ODP) library:
 
     git clone https://git.linaro.org/lng/odp.git
     cd odp
-    git checkout v1.19.0.2
+    git checkout v1.35.0.0
```

Comment:
Old Linaro URL used above. There seems to be several of these old links in the code base. Could be fixed in a separate commit.